### PR TITLE
feat: add link support for discussion

### PR DESCRIPTION
**Pull request by bxb100**
source: https://github.com/gitify-app/gitify/pull/538

Co-authored-by: Araxeus
## reference link
* https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions#search
* https://docs.github.com/en/search-github/searching-on-github/searching-discussions

### CODE REVIEW

**Review by colinlienard — APPROVED** [review]
LGTM, thanks a lot! 
It might be great to also show the comment and name of the persons who commented, I could do it in another pr

**Comment by bxb100** [comment]
Thks review, I can do it in this PR, so give me some time

**Comment by bxb100** [comment]
done @colinlienard

File: src/lib/components/dashboard/notifications/NotificationDescription.svelte [hunk 1]
```diff
@@ -72,6 +72,7 @@
 	.description {
 		@include typography.base;
 
+		hyphens: auto;
```
Comment (colinlienard) [inline src/lib/components/dashboard/notifications/NotificationDescription.svelte]: +1, didn't know that was a thing

**Review by colinlienard — APPROVED** [review]
LGTM, thanks a lot again for your work!

**Pull request merged**